services/catarse/lib/tasks/dispatch_project_metric_storage_refresh_task.rake

```diff
@@ -24,7 +24,7 @@ class DispatchProjectMetricStorageRefreshTask
     ProjectMetricStorage
       .joins(:project)
       .where("refreshed_at < now() - '5 minutes'::interval")
-      .where(projects: {state: 'online'})
+      .where(projects: {state: [:online, :waiting_funds, :failed, :successful]})
```

Comment:
**Cosmético:**
Nas próximas vezes, quando for criar um array de symbols, pode fazer assim:
```suggestion
      .where(projects: { state: %i[online waiting_funds failed successful] })
```